cli/Cargo.toml

```diff
@@ -22,7 +22,7 @@ unexpected_cfgs = { level = "warn", check-cfg = ['cfg(test)'] }
 [dependencies]
 anyhow = "1.0"
 atree = "0.5.2"
-c2pa = { workspace = true , features = [
```

Comment:
Actually I didn't intend to sync c2patool with this change yet.

Reply:
I suppose it doesn't hurt but it could be confusing if we don't keep everything in sync. Won't the other crates have the same issue?

Reply:
It looks like the cargo team doesn't plan to support this syntax for `cargo publish`: https://github.com/rust-lang/cargo/issues/11133

Reply:
Someone else on that thread resolved the issue by moving to workspace packaging
https://github.com/boa-dev/boa/pull/2297